bin/cmd/batch/csv.js

```diff
@@ -85,3 +96,16 @@ function generateTemplates (argv) {
 
   return templates
 }
+
+function generateFields (argv) {
+  // cast scalar (single flag specified) values to arrays
+  const c = _.castArray(argv.column)
+  const s = _.castArray(argv.selector)
+
+  // report error if pairs are unbalanced
+  if (_.size(c) !== _.size(s)) {
+    throw new Error('error: you pair every -c (column) flag with a -s (selector) flag.')
```

Comment:
This message was copied from above, but I think both could be improved by adding the word 'must'